rooms.json5

```diff
@@ -0,0 +1,112 @@
+                "121401",   # michaelpri
+                "116218",   # JamesENL
+                "82927",    # Braiam
+                "11606",    # bwDraco
+                "19761",    # Ilmari Karonen
+                "108271",   # Andrew T.
+                "171054",   # Magisch
+                "190011",   # Petter Friberg
+                "165661",   # Tunaki
+                "145086",   # Wai Ha Lee
+                "137665",   # ByteCommander
+                "147884",   # wythagoras
+                "186395",   # Åna
+                "181293",   # Ashish Ahuja
+                "163686",   # Gothdo
+                "145827",   # angussidney
+                "244748",   # Supreme Leader SnokeDetector (angussidney's sock)
+                "121520",   # ArtOfCode
+                "244382",   # Lt. A. Code (ArtOfCode's sock to test things with)
+                "137388",   # QPaysTaxes
+                "212311",   # Ryan Bemrose
+                "172397",   # Kyll
+                "224538",   # FrankerZ
+                "61202",    # OldSkool
+                "56166",    # Jan Dvorak
+                "133966",   # DavidPostill
+                "22839",    # djsmiley2k
+                "97389",    # Kaz Wolfe
+                "144962",   # DJMcMayhem
+                "139423",   # NobodyNada
+                "62118",    # tripleee
+                "130558",   # Registered User
+                "128113",   # arda
+                "164318",   # Glorfindel
+                "175347",   # Floern
+                "180274",   # Alexander O'Mara
+                "158742",   # Rob
+                "207356",   # 4castle
+                "133031",   # Mithrandir
+                "215671",   # Locutus of Borg (Mithrandir's Sock)
+                "169713",   # Mego
+                "126657",   # Cerbrus
+                "10145",    # Thomas Ward
+                "161943",   # J F
+                "195967",   # CaffeineAddiction
+                "5363",     # Stijn
+                "248139",   # FelixSFD
+                "156721",   # D-side
+                "167070",   # quartata
+                "172450",   # Hovercraft Full Of Eels
+                "56200",    # Eric Leschinski
+                "211021",   # Henders
+                "255290",   # Gypsy Spellweaver
+                "64521",    # CalvT
+                "165474",   # Hyper Neutrino
+                "281362",   # Hyper Neutrino v2
+                "169252",   # Cai
+                "155243",   # Nisse Engström
+                "69330",    # Sconibulus
+                "164187",   # Okx
+                "202619",   # John Militer
+                "262693",   # suraj
+                "11287",    # Martin Sleziak
+                "88588",    # NVZ
+                "281674",   # paper1111
+                "279119",   # Tetsuya Yamamoto
+                "307652",   # Ajay Brahmakshatriya
+                "238145",   # Owen Hines
+                "268731"    # Sagar V
```

Comment:
If you put a trailing comma here, it will make diffs prettier when we add someone new.